### PR TITLE
Use GitHub Actions and Fix Client Build

### PR DESCRIPTION
**Pull request by bash**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  server:
+    name: Server
+    runs-on: ubuntu-latest
```
Comment (jeremystucki) [inline .github/workflows/build.yml:12]: Maybe use a fixed ubuntu version

Comment (jeremystucki) [inline .github/workflows/build.yml:12]: Maybe use a fixed ubuntu version

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    steps:
+      - uses: actions/checkout@v1
+      - name: Run Tests
+        run: ./ci/test_server.sh
+      - uses: codecov/codecov-action@v1
+        with:
+          file: ./lcov.info
+  client:
+    name: Client
+    runs-on: ubuntu-latest
```
Comment (jeremystucki) [inline .github/workflows/build.yml:25]: Same

File: .github/workflows/build.yml (hunk 1 above)
Comment (bash) [reply .github/workflows/build.yml]: b98eb22

File: .github/workflows/build.yml (hunk 2 above)
Comment (bash) [reply .github/workflows/build.yml]: b98eb22

**Review by jeremystucki — APPROVED** [review]

**Pull request merged**